docs/DeveloperGuide.md

```diff
@@ -220,92 +222,57 @@ The following activity diagram shows the process when a user calls the `EditProf
-
-![UndoRedoState2](images/UndoRedoState2.png)
+#### Description
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** If a command fails its execution, it will not call `Model#commitAddressBook()`, so the address book state will not be saved into the `addressBookStateList`.
+In this section, we will describe how the `AddAttendeesCommand` is implemented. `AddAttendeesCommand` is used to add existing `Profile`s to an `Event` as attendees.
 
-</div>
+The `AddAttendeesCommand` class extends the `EventCommand` abstract class. `EventCommand` is an abstract class which extends the `Command` class. `AddAttendeesCommand` overrides the `Command#execute` method to add new attendees to an event when called.
 
-Step 4. The user now decides that adding the person was a mistake, and decides to undo that action by executing the `undo` command. The `undo` command will call `Model#undoAddressBook()`, which will shift the `currentStatePointer` once to the left, pointing it to the previous address book state, and restores the address book to that state.
+#### Proposed implementation
 
-![UndoRedoState3](images/UndoRedoState3.png)
+Similar to other commands, the user input to add attendees is passed to `LogicManager` to be executed. The following details the key difference in the methods that is invoked when the `AddAttendeesCommand#execute()` method is called:
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** If the `currentStatePointer` is at index 0, pointing to the initial AddressBook state, then there are no previous AddressBook states to restore. The `undo` command uses `Model#canUndoAddressBook()` to check if this is the case. If so, it will return an error to the user rather
-than attempting to perform the undo.
+* Model#updateEventAttendees() - Adds the specified `Profile`s to the `Attendees` attribute of the `Event`.
 
-</div>
+Given below is an example usage scenario on how `AddAttendeesCommand` can be used and how it works.
 
-The following sequence diagram shows how the undo operation works:
+Step 1. The user clears all the data in the application with the `ClearCommand`and then enters the following commands:
 
-![UndoSequenceDiagram](images/UndoSequenceDiagram.png)
+* `profile -a n/John ...` - adds a `Profile` with the name "John"
+* `event -a n/Consultation ...` - adds an `Event` with the title "Consultation"
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `UndoCommand` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
+![AddAttendeeState0](images/AddAttendeesState0.png)
 
-</div>
+Step 2. The user executes `event -ap 1 n/1` to add the `Profile` John to the `Attendees` attribute under the `Event` Consultation.
```

Comment:
Perhaps you could explain what the `1` here means? But that said, I'm not sure how much implementation details we need here since they're all "proposed" features